lib/appstate.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class AppState {
+  var nickName = ValueNotifier<String>('Yanciman');
```

Comment:
You could implement setters for these, so `state.nickName = 'robinlinguini';` will set the value of the ValueNotifier?

Not sure if it's too magic, but it seems to be how things are done in Flutter.

See: https://dart.dev/guides/language/language-tour#getters-and-setters

Reply:
Magic indeed